server/gamemodes/chess.py

```diff
@@ -271,14 +273,19 @@ def validate_king_move(
             if (not rook) or rook.has_moved:
                 return False
             for empty_file in empty_files:
-                if not self.get_piece(rank, empty_file):
+                if self.get_piece(rank, empty_file):
                     return False
-            return not self.hypothetical_check(
-                king.side, (king, rank, file), (rook, rank, rook_end)
-            )
+            if self.hypothetical_check(king.side):
+                return False
+            if self.hypothetical_check(king.side,(king,rank,rook_end)):
```

Comment:
```suggestion
            if self.hypothetical_check(king.side, (king, rank, rook_end)):
```
Lint!

Reply:
Sorry about this, I thought Visual Studio 2019 was linting for me.  I may need to move to VS Code as I can't seem to figure out how to lint with VS2019.